### ISSUE TITLE
Host / access key / secret key deprecated + Gradle compatibility issues

### ISSUE THREAD

**Issue by mregnauld** [nrubin29/flutter_acrcloud#6]
Thanks for sharing, and for your Flutter package!
That being said, it looks like it's not usable anymore. I have Gradle plugin compatibility issues, and the "host / access key / secret key" seems to be deprecated, in favor of the Personal Access Token.
Do you plan to update your package soon?

**Comment by nrubin29** [comment]
Hi,

Could you tell me where you saw that the host / access key / secret key scheme is deprecated? I just looked at both the iOS and Android SDKs and they both use them.

Also, could you tell me what the Gradle error is?

Thanks.

**Comment by mregnauld** [comment]
Sorry for the late answer.

So I tried again with the last version (0.2.0), and I get the following error when I try to run the test app on an Android device:

```
The Android Gradle plugin supports only Kotlin Gradle plugin version 1.5.20 and higher.
The following dependencies do not satisfy the required version:
project ':flutter_acrcloud' -> org.jetbrains.kotlin:kotlin-gradle-plugin:1.3.50
Update plugins
Affected Modules: app
```

**Comment by nrubin29** [comment]
I just released version 0.2.1 where I bumped the Kotlin and Gradle versions. Give it a try and see if it fixes your problem.

**Comment by mregnauld** [comment]
Now it compiles and run!
